mails/themes/index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
```

Comment:
all headers are in 2018

Reply:
not ALL of them, don't be mean!
but some remained ^^ fixed